jade/distribution.yaml

```diff
@@ -3650,6 +3650,51 @@ repositories:
       url: https://github.com/ros-visualization/rqt_robot_plugins.git
       version: master
     status: developed
+  rsv_balance:
+    doc:
+      type: git
+      url: https://github.com/robosavvy/rsv_balance.git
+      version: master
+    release:
+      packages:
+      - rsv_balance_description
+      - rsv_balance_msgs
+      url: https://github.com/robosavvy/rsv_balance.git
```

Comment:
This is not a valid `release` repository (same fore the other release entries). Please read the bloom tutorial on how to release a ROS package: http://wiki.ros.org/bloom